ludwig/encoders/text_encoders.py

```diff
@@ -50,16 +51,34 @@ class BERTEncoder(TextEncoder):
 
     def __init__(
             self,
-            pretrained_model_name_or_path='bert-base-uncased',
-            reduce_output='cls_pooled',
-            trainable=True,
-            num_tokens=None,
-            max_sequence_length=None,
+            use_pretrained: bool = True,
+            pretrained_model_name_or_path: str = 'bert-base-uncased',
+            trainable: bool = True,
+            reduce_output: str ='cls_pooled',
+            num_tokens: Optional[int] = None,
+            max_sequence_length: Optional[int] = None,
+            vocab_size: Optional[int] = 30522,
```

Comment:
Can we consolidate `num_tokens` and `vocab_size` into one singel parameter? I beleive they have similar goals.